app/src/main/res/layout/nav_section_item.xml

```diff
@@ -1,16 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="16dp"
+    android:layout_height="wrap_content"
     android:background="@android:color/transparent"
     android:orientation="vertical"
     android:importantForAccessibility="no">
 
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:layout_centerVertical="true"
```

Comment:
I think it would be good to add a bit of vertical margin to this view. Changing its parent to wrap content removes the space between the separator and menu items.

Reply:
I added `paddingTop` and `paddingBottom` to the parent LinearLayout